from controllers.utils.error_handling import handle_404, handle_405

def handle_error_404(e):
    print(e)
    return handle_error_404(e)

def handle_error_405(e):
    return handle_error_405(e)